api-reference/50 Common/Object Structures/positionConfig/at/at.md

```diff
@@ -5,26 +5,113 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-The target element position that the widget is positioned against.
+Specifies a side or corner of the [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) at which the overlay element should be positioned. 
 
 ---
-The **at** option can take on an object containing the **x** and **y** fields, which specify horizontal and vertical position specifier respectively, or a string value consisting of horizontal and vertical position specifiers separated by a space (e.g., "left top"). The default value for each position specifier is "center". If you assign the "left" value to this option, it will be converted to the "left center" value.
 
-    <!--JavaScript-->
-    position: { at: 'left bottom' };
+To set this option, use an object with the **x** and **y** fields or one of the string shortcuts listed in the accepted values.
```

Comment:
Can you be more specific than "use an object with the..."?
For example, do you mean:
To set this option, specify the object's **x** and **y** fields, or use a string shortcut from the accepted values list.

If not, please suggest a revised version.

Reply:
Did you see my comment above?